src/tentacles/core.clj

```diff
@@ -91,7 +91,8 @@
                     {:keys [auth throw-exceptions follow-redirects accept
                             oauth-token etag if-modified-since user-agent
                             otp]
-                     :or {follow-redirects true throw-exceptions false}
+                     :or {follow-redirects (or (:follow-redirects defaults) true)
+                          throw-exceptions (or (:throw-exceptions defaults false))}
```

Comment:
I think `or` here is unnecessary, and it could be removed from the line above if the `true` is moved into the default position.